src/mesh/boundary_info.C

```diff
@@ -1285,6 +1285,37 @@ void BoundaryInfo::raw_boundary_ids (const Elem * const elem,
 
 
 
+void BoundaryInfo::copy_boundary_ids
+  (const BoundaryInfo &old_boundary_info,
```

Comment:
Space on both sides of ampersand, also this will be reindented by the reindent script.